### PR TITLE
[SensitiveContentAnalysis] Add support for Xcode 15

### PR DESCRIPTION
**Pull request by haritha-mohan**

### CODE REVIEW

File: src/sensitivecontentanalysis.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+	[NoTV, NoWatch, Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface SCSensitivityAnalyzer {
+		[Export ("analysisPolicy", ArgumentSemantic.Assign)]
+		SCSensitivityAnalysisPolicy AnalysisPolicy { get; }
+
+		[Export ("analyzeImageFile:completionHandler:")]
+		[Async]
+		void AnalyzeImage (NSUrl fileURL, Action<SCSensitivityAnalysis, NSError> completionHandler);
```
Comment (dalexsoto) [inline src/sensitivecontentanalysis.cs:37]:
```suggestion
		void AnalyzeImage (NSUrl fileUrl, Action<SCSensitivityAnalysis, NSError> completionHandler);
```

File: src/sensitivecontentanalysis.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+		[Export ("analyzeCGImage:completionHandler:")]
+		[Async]
+		void AnalyzeImage (CGImage image, Action<SCSensitivityAnalysis, NSError> completionHandler);
+
+		[Export ("analyzeVideoFile:completionHandler:")]
+		[Async]
+		NSProgress AnalyzeVideo (NSUrl fileURL, Action<SCSensitivityAnalysis, NSError> completionHandler);
```
Comment (dalexsoto) [inline src/sensitivecontentanalysis.cs:45]:
```suggestion
		NSProgress AnalyzeVideo (NSUrl fileUrl, Action<SCSensitivityAnalysis, NSError> completionHandler);
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — APPROVED** [review]

File: src/sensitivecontentanalysis.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 Xamarin Inc. All rights reserved.
```
Comment (mandel-macaque) [inline src/sensitivecontentanalysis.cs:1]: @haritha-mohan is back to the future ;)

**Review by mandel-macaque — APPROVED** [review]
@haritha-mohan lets fix the copyright, or not even add it :)

**Review by stephen-hawley — APPROVED** [review]

File: src/sensitivecontentanalysis.cs (hunk 3 above)
Comment (haritha-mohan) [reply src/sensitivecontentanalysis.cs]: lol fixed

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**